app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +240,18 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      {if @state.workflows.length > 0
+        meta = @state.workflows[0].getMeta()
+
+        <Paginator
+          className="lab-visibility-paginator"
```

Comment:
I wasn't clear, but what I meant is adding the `paginator` class as defined in this [file](https://github.com/zooniverse/Panoptes-Front-End/blob/master/css/paginator.styl) and then modifying in a new sub-class defined in the lab file if needed. You shouldn't need to copy and paste over the styles from the original talk class. 

The CSS in this project is a bit of a mess, but we're trying to use BEM for new classes, so try to follow that if you can and it makes sense for the changes.